### ISSUE TITLE
Changelog miss some storyboards settings

### ISSUE THREAD

**Issue by weber-s** [Chocobozzz/PeerTube#6329]
### Describe the problem

Hello!

I spotted some missing storyboard related settings in the changelog for the v6.0.0.

- The `storage.storyboards` entry must be provided
- The `cache.storyboards` entry must be provided

Further, the `storyboards` entry is not in the `default.yaml` file generated during update.

Hope it help others!

Cheers,

### Additional information

* PeerTube instance URL: https://video.gresille.org
* PeerTube version: 6.0.4
* NodeJS version: 18
* Ffmpeg version: 5.1.4
* Redis version:
* PostgreSQL version: 15

**Comment by Chocobozzz** [comment]
Hi,

> The storage.storyboards entry must be provided

It is :thinking: https://github.com/Chocobozzz/PeerTube/releases/tag/v6.0.0

> The cache.storyboards entry must be provided

We do not notify about every key to be added in the configuration file, just the important ones because...

> Further, the storyboards entry is not in the default.yaml file generated during update.

...these keys are present in `default.yaml`: https://github.com/Chocobozzz/PeerTube/blob/v6.0.0/config/default.yaml#L435. How do you upgrade peertube?

**Comment by weber-s** [comment]
Well... apparently I was a bit tired yesterday... Sorry.

I upgraded following the doc, but checked the diff between `config/production.yml.new` and `config/production.yml` (the `.new` one was missing keys).


For the missing `storyboards` keys in yaml, I recheck, and it is not present in the `default.yaml` I have.
I mannualy added:

```
storyboards:
  # Generate storyboards of local videos using ffmpeg so users can see the video preview in the player while scrubbing the video
  enabled: true
```

**Comment by Chocobozzz** [comment]
We had a bug with the `production.yaml.new` generation that should be fixed in next release.

However the upgrade script should upgrade `default.yaml` with the latest changes: https://github.com/Chocobozzz/PeerTube/blob/develop/server/scripts/upgrade.sh#L89

**Comment by weber-s** [comment]
Ok!

However, I do not have the `storyboards` entry at the end of the default.yaml, although it is the 6.0.4 version:

```
peertube@drosophile:~/versions/peertube-v6.0.4$ tail config/default.yaml 
    resumable_upload:
      # Max size of upload chunks, e.g. '90MB'
      # If null, it will be calculated based on network speed
      max_chunk_size: null

  menu:
    login:
      # If you enable only one external auth plugin
      # You can automatically redirect your users on this external platform when they click on the login button
      redirect_on_single_external_auth: false
peertube@drosophile:~/versions/peertube-v6.0.4$ 
```

Which is... strange ?

But no big deal, I let it here for the record if someone search it on the internet some day.

**Comment by Chocobozzz** [comment]
> However, I do not have the storyboards entry at the end of the default.yaml, although it is the 6.0.4 version:

New keys are not necessarily at the end of the file. Use `grep 'storyboard' config/default.yaml` instead

**Comment by weber-s** [comment]
Same. 

```
peertube@drosophile:~/versions/peertube-v6.0.4$ grep storyboards config/default.yaml 
  storyboards: 'storage/storyboards/'
  storyboards:
    size: 500 # Max number of video storyboards you want to cache
peertube@drosophile:~/versions/peertube-v6.0.4$ 
```

The `storyboards.enable` key is missing.

cf: https://github.com/Chocobozzz/PeerTube/blob/v6.0.4/config/default.yaml

It is on develop however : https://github.com/Chocobozzz/PeerTube/blob/develop/config/default.yaml and the blame say it should be on 6.0.4 . The release seems to miss it.

**Comment by Chocobozzz** [comment]
Seems I'm also tired ^^ ability to disable storyboard generation will be available in next peertube version (6.1) :)
